tests/unit/specs/components/LocalesMenu.spec.js

```diff
@@ -5,30 +5,30 @@ import LocalesMenu from '@/components/LocalesMenu'
 import { Core } from '@/core'
 
 describe('LocalesMenu', () => {
-  const { localVue, i18n } = Core.init(createLocalVue()).useAll()
+  const core = Core.init(createLocalVue()).useAll()
+  const { localVue, i18n } = core
   let wrapper = null
 
-  afterEach(() => {
-    localStorage.removeItem('locale')
-    i18n.locale = 'en'
+  beforeEach(async () => {
+    await core.loadI18Locale('en')
   })
 
```

Comment:
Maybe adding an afterAll with `await core.loadI18Locale('en')` would enforce a teardown without side effects on the locale.

Reply:
Good call too!